### DISCUSSION TITLE
Is IOS build free from trackers?

### DISCUSSION THREAD

**Discussion by macruspareto** [organicmaps/organicmaps#5129] [Q&A]
Hello everyone!

Just a quick question: 

Is IOS version of Organic Maps free from trackers and 'phoning-home' telemetry?

I'm using OM for Android (the F-Droid version of it) and it seems to be clean. Knowing that IOS-based developers prefer to stuff any possible tracker into their codebase, I'm a little bit worried.

Thanks !

**Comment by biodranik** [comment]
Any official Organic Maps version (that is built and distributed by us, maintainers), including iOS, is completely free from any trackers. We use the same source code everywhere.
